### ISSUE TITLE
Navbar design suggestion

### ISSUE THREAD

**Issue by TheCoward** [Retrospring/bugs#16]
Yet another design suggestion from me! Yes, I know you don't like them that much :^)
Either way, that can stay as a userstyle thing, but I've been messing around with hover.css, and see, I made something with the navbar, which you can see on [this webm](http://a.pomf.se/fdkuuh.webm). Yes, I know it doesn't look that good on the profile picture, and I'm trying to get rid of the persistent shadow, but it looks fine, doesn't it? Remains to know if you'd like to implement it or not.

Here's teh code, and yes, it's that fucking hardcoded:
```css
/* Trying something */

.nav>li {
    -webkit-transform: translateZ(0);
    transform: translateZ(0);
    -webkit-backface-visibility: hidden;
    backface-visibility: hidden;
    -moz-osx-font-smoothing: grayscale;
    position: relative;
    overflow: hidden;
}
.nav>li:before {
    content: "";
    position: absolute;
    z-index: -1;
    left: 0;
    right: 0;
    bottom: 0;
    background: #fff;
    height: 4px;
    -webkit-transform: translateY(4px);
    transform: translateY(4px);
    -webkit-transition-property: transform;
    transition-property: transform;
    -webkit-transition-duration: 0.3s;
    transition-duration: 0.3s;
    -webkit-transition-timing-function: ease-out;
    transition-timing-function: ease-out;
}
.nav>li:hover:before,
.nav>li:focus:before,
.nav>li:active:before {
    -webkit-transform: translateY(0);
    transform: translateY(0);
}
.navbar-inverse .btn-link:hover,
.navbar-inverse .btn-link:focus {
    color: #fff;
}
.navbar-inverse .navbar-nav>li>a:hover,
.navbar-inverse .navbar-nav>li>a:focus {
    color: #fff;
}
.navbar-nav>li>a {
    color: #fff;
}
```
Apparently it also breaks the dropdown, but guess I only need to remove the :focus.
![crap](https://a.pomf.se/zwstsn.png)

**Comment by yukimono** [comment]
i'll see what i can retrofit onto the stylesheets

**Comment by yukimono** [comment]
You actually didn't need to fiddle with transforms, just modifying height did the trick.

**Comment by thebluecorsair** [comment]
I like the design changes personally for the navigation links, however I don't exactly like the circular avatar concept for it.

**Comment by nilsding** [comment]
Seems like not everyone likes circular avatars! (and I'm one of them)

The navbar changes seem nice though.  I would disable them for mobile viewports though. 
![wp_ss_20150422_0001](https://cloud.githubusercontent.com/assets/1809170/7264189/723153ec-e889-11e4-9a25-1b31423aaa5c.png)

**Comment by yukimono** [comment]
Mmmm, yeah i have to do that "orz

**Comment by yukimono** [comment]
Changes are made.

![screen shot 2015-04-22 at 7 23 29](https://cloud.githubusercontent.com/assets/614231/7265982/64a60f10-e8c0-11e4-9240-8f02c1826c8c.png)
![screen shot 2015-04-22 at 7 23 36](https://cloud.githubusercontent.com/assets/614231/7265983/64a95760-e8c0-11e4-93c9-b425fd81e322.png)

I also added in some consistency for the mobile navbar so everything is equally spaced out

**Comment by thebluecorsair** [comment]
Looking good, nice changes.

**Comment by TheCoward** [comment]
@TheBlueCorsair Rounded avatars were meant for my userstyle, nothing else.
@yukimono Well, like I said, I messed with hover.css but I'm no good with
animations.
@nilsding Yeah, it was my first concern.

Closed #16 <https://github.com/Retrospring/bugs/issues/16> via
Retrospring/retrospring@1600cdf
<https://github.com/Retrospring/retrospring/commit/1600cdf2954fce8626039be9d5b257a3af07dbcb>
.

—
Reply to this email directly or view it on GitHub
<https://github.com/Retrospring/bugs/issues/16#event-286892855>.
